src/scripts/pbs_status.py

```diff
@@ -358,18 +358,17 @@ def get_qstat_location():
     if _qstat_location_cache != None:
         return _qstat_location_cache
 
-    try:
-        location = os.path.join(config.get('pbs_binpath'), 'qstat')
-    except KeyError:
-        cmd = 'which qstat'
-        child_stdout = os.popen(cmd)
-        output = child_stdout.read()
-        location = output.split("\n")[0].strip()
-        if child_stdout.close():
-            raise Exception("Unable to determine qstat location: %s" % output)
-
-    _qstat_location_cache = location
-    return location
+    if not (config.has_option('pbs_binpath') and config.get('pbs_binpath')):
+        config.set('pbs_binpath', '/usr/bin')
+    cmd = 'echo "%s/%s"' % (config.get('pbs_binpath'), 'qstat')
```

Comment:
Not calling `which` every time would improve scalability. If the user really wants it, they can still specify `which` in the config.